Gruntfile.js

```diff
@@ -18,22 +20,29 @@ module.exports = (grunt) => {
   const TARGET_MIN = 'vexflow-min.js';
 
   // Used for eslint and docco
-  const SOURCES = ['src/*.ts', 'src/*.js', '!src/header.js'];
+  const SOURCES = ['./src/*.ts', './src/*.js', '!./src/header.js'];
 
-  // Take all test files in 'tests/' and build TARGET_TESTS
-  const TARGET_TESTS = path.join(BUILD_DIR, 'vexflow-tests.js');
-  const TEST_SOURCES = ['tests/vexflow_test_helpers.js', 'tests/mocks.js', 'tests/*_tests.js', 'tests/run.js'];
+  // Take all test files in 'tests/' and build TARGET_TESTS_BROWSER
+  const TARGET_TESTS_BROWSER = 'vexflow-tests.js';
```

Comment:
In this case, maybe we should be clearer: `vexflow-tests-browser.js` and `vexflow-tests.node.js`.

Reply:
I will try node bundle in the browser, this might solve the problem with 2 files. If no, I will change the names accordingly.

Reply:
That would be great. Alternatively, we can also trying to build just one `vexflow-tests.js` and adding a small shim for the node version.

Reply:
I tried using node bundle and it did not work. I will try a node shim though.